src/libraries/Microsoft.PowerFx.Core/Public/Config/SymbolTableOverRecordType.cs

```diff
@@ -25,17 +26,14 @@ public SymbolTableOverRecordType(RecordType type, ReadOnlySymbolTable parent = n
 
         // Key is the logical name. 
         // Display names are in the NameLookupInfo.DisplayName field.
-        IReadOnlyDictionary<string, NameLookupInfo> IGlobalSymbolNameResolver.GlobalSymbols
+        IEnumerable<KeyValuePair<string, NameLookupInfo>> IGlobalSymbolNameResolver.GlobalSymbols            
         {
             get
-            {
-                var map = new Dictionary<string, NameLookupInfo>();
```

Comment:
>map [](http://example.com/codeflow?start=20&length=3)

yes! This is a win - composing gets simpler - we don't need to create a temp dictionary.